src/settings-ui/Settings.UI/SettingsXAML/Views/GeneralPage.xaml

```diff
@@ -175,6 +175,15 @@
                         </InfoBar.ActionButton>
                     </InfoBar>
 
+                    <InfoBar
+                        x:Uid="GPO_PeriodicUpdateCheckIsDisabled"
+                        Margin="0,-10,0,10"
+                        IsClosable="False"
+                        IsOpen="{x:Bind Mode=OneWay, Path=ViewModel.ShowPeriodicUpdateCheckGpoInformation}"
+                        IsTabStop="{x:Bind Mode=OneWay, Path=ViewModel.ShowPeriodicUpdateCheckGpoInformation}"
+                        Severity="Warning"
+                        Visibility="{Binding Mode=OneWay, Path=ViewModel.ShowPeriodicUpdateCheckGpoInformation, Converter={StaticResource BoolToVisibilityConverter}}" />
```

Comment:
Please remove the Visibility property. It has no use for an InfoBar.
And is the Margin needed?

Reply:
The margin is a layout fix. And as I am not sure if the margin not leads to a strange layout if the Infobar is closed, I have added the visibility property.

Reply:
@Jay-o-way
I have removed the obsolete visibility property. Is has definitely no use.

Regarding the margin fix: We need it and it only moves the space from before the info bar to after it. I have made two shots. The first one without margin fix and the second with margin fix:

![image](https://github.com/microsoft/PowerToys/assets/61519853/f2b1e0cd-5612-450b-863d-d69bb456b704)

![image](https://github.com/microsoft/PowerToys/assets/61519853/68d5052b-d189-4c62-be8a-7b43111f528d)